docs/pizza_completion.md

```diff
@@ -17,7 +17,7 @@ See each sub-command's help for details on how to use the generated script.
 ### Options inherited from parent commands
 
 ```
-  -c, --config string       The saucectl config (default ".sauced.yaml")
+  -c, --config string       The codeowners config (default "~/.sauced.yaml")
```

Comment:
The default is `.sauced.yaml`, not the root of the user's home folder, i.e `~/.sauced.yaml`. Here and in the rest of the docs.

```suggestion
  -c, --config string       The codeowners config (default ".sauced.yaml")
```

Reply:
done!